CITATION.cff

```diff
@@ -158,6 +158,8 @@ authors:
     - given-names: Fesselier
       family-names: Laetitia
       affiliation: 'McGill Centre for Integrative Neuroscience, McGill University, Montreal, Canada'
+    - given-names: Jonathan
+      family-names: Vanhoecke
```

Comment:
@JonathanVHoecke we forgot to add you to this list earlier when you made your first PR! Can you please tell us your affiliation and ORCID?

Same goes for @richardkoehler -- we still need your affiliation and ORCID

Thanks!

Reply:
@sappelhoff
Sorry, I must have overlooked that so thanks for the notice! 

I am currently listed under:     

      - given-names: Richard
        family-names: Koehler
        orcid: 'https://orcid.org/0000-0002-5219-1289'
       
The ORCID is correct, but if possible it would be kind if you could modify given and family-names and add the affiliation like below (changed fields in bold). Or should I add the changes in a PR myself? Thanks in advance!

      - given-names: Richard M.
        family-names: Köhler
        orcid: 'https://orcid.org/0000-0002-5219-1289'
        affilitation: Movement Disorder and Neuromodulation Unit, Department of Neurology, Charité – Universitätsmedizin Berlin, Germany

Reply:
Thank you so much @sappelhoff . Yes  we will do. I will also get back on de PR about sessions.tsv as I unfortunately did not manage before my holidays.
Many thanks again, bw
Jonathan

Reply:
> I will also get back on de PR about sessions.tsv

That one has already been merged 😅